config/data/canonical_states.yml

```diff
@@ -0,0 +1,43 @@
+India:
+  - Andhra Pradesh
+  - Arunachal Pradesh
+  - Assam
+  - Bihar
+  - Chhattisgarh
+  - Goa
+  - Gujarat
+  - Haryana
+  - Himachal Pradesh
+  - Jharkhand
+  - Karnataka
+  - Kerala
+  - Madhya Pradesh
+  - Maharashtra
+  - Manipur
+  - Meghalaya
+  - Mizoram
+  - Nagaland
+  - Odisha
+  - Punjab
+  - Rajasthan
+  - Sikkim
+  - Tamil Nadu
+  - Telangana
+  - Tripura
+  - Uttar Pradesh
+  - Uttarakhand
+  - West Bengal
+Bangladesh:
+  - Barisal
+  - Bogura
+  - Chittagong
+  - Comilla
+  - Dhaka North
+  - Dhaka South
+  - Gazipur
+  - Khulna
+  - Mymensingh
+  - Narayanganj
+  - Rajshahi
+  - Rangpur
+  - Sylhet
```

Comment:
This does not have Ethiopia states currently. We can run Region Backfill + enable flipper on ET once we have that information.

Reply:
These are the Ethiopia "states" - there are 12 of them.  In Ethiopia they are known as "regions": ..."There are currently ten regional states and two chartered cities".

https://en.wikipedia.org/wiki/Regions_of_Ethiopia#List_of_regions_and_city_administrations

I don't think there is a reason to not add them right away?

Reply:
The CSV that we received has names in the amharic script, not sure if we're planning on using english or amharic. I'll ask for this on slack. Meanwhile the state names are not a blocker, I meant.